scribble/db/schema.rb

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS posts;
+DROP TABLE IF EXISTS comments;
+
+CREATE TABLE posts (
+  id SERIAL PRIMARY KEY,
+  email TEXT
+);
+
+CREATE TABLE comments (
+  id SERIAL PRIMARY KEY,
+  email TEXT,
+  post_id INTEGER
+
+);
```

Comment:
Echoing my comments left in the `seeds.rb` file. This shouldn't contain SQL. Instead, it will contain Ruby code that generates tables. You will never modify the `schema.rb` file yourself. You should always use and run migrations, which will define what your schema file looks like.